pkg/member/member_control.go

```diff
@@ -87,7 +89,16 @@ func NewMemberControl(etcdConnConfig *brtypes.EtcdConnectionConfig) Control {
 }
 
 // AddMemberAsLearner add a member as a learner to the etcd cluster
-func (m *memberControl) AddMemberAsLearner(ctx context.Context) error {
+func (m *memberControl) AddMemberAsLearner(ctx context.Context, dataDir string) error {
```

Comment:
`AddMemberAsLearner` is a low level interface and should not do the check for data dir. This should be done by the caller.

Reply:
make sense, I have addressed it. PTAL